src/utils/Util.ts

```diff
@@ -307,3 +307,19 @@ export function checkIfNameChanged() {
         return false;
     }
 }
+
+// reads data from local file into a JSON object
+export function getFileDataAsJson(file: string): any {
```

Comment:
this is all awesome but I noticed at least one expecting an array. how about adding the default result as an argument (file:string, default:any = null) so that if there is an issue the caller can get it and still use it without failing. there's one call that expects ".logs" and if it returns null, that will throw a null pointer.